lib/driver.js

```diff
@@ -240,13 +240,20 @@ class WebDriverAgentDriver extends BaseDriver {
            !_.includes(commandsToNotProxyToSafari, command);
   }
 
-  async executeCommand (cmd) {
+  async executeCommand (cmd, ...args) {
     if (this.shouldProxyToSafari(cmd)) {
       let safariDriver = await this.getSafariDriver();
-      return await safariDriver.executeCommand(...arguments);
+      return await safariDriver.executeCommand(...args);
```

Comment:
The previous solution use `arguments`, which is _all_ the arguments. This new `args` is everything except `cmd`, which is separated out.